core/src/main/kotlin/com/justai/jaicf/api/BotRequest.kt

```diff
@@ -28,9 +28,26 @@ enum class BotRequestType {
  */
 open class QueryBotRequest(
     override val clientId: String,
+    input: String
+) : BotRequest {
+
+    private var inputValue = input
```

Comment:
Why not var with private set?

Reply:
private setters are not allowed for open properties